packages/aws-rfdk/docs/upgrade/upgrading-1.5.md.yaml

```
@@ -0,0 +1,13 @@
+# Upgrading to RFDK v1.5.x or Newer
+
+## Updating DocumentDB
+
+Please backup your existing database before upgrading to RFDK 1.5.x.
+
+The default DocumentDB engine version used by RFDK versions >= 1.5.x have been upgraded to 5.0.0. Note that if you have an existing DocumentDB cluster with previous versions, you will need to manually upgrade by following the documentation: https://docs.aws.amazon.com/documentdb/latest/developerguide/docdb-mvu.html
```

```suggestion
In RFDK versions >= 1.5.x, we upgraded the default DocumentDB engine version to 5.0.0. Note that if you have an existing DocumentDB cluster with an older version, you will need to manually upgrade by following the documentation: https://docs.aws.amazon.com/documentdb/latest/developerguide/docdb-mvu.html
```

Some nitpicks, and consider changing to use active voice.